packages/vuetify/src/labs/VTimePicker/VTimePicker.tsx

```diff
@@ -264,8 +264,8 @@ export const VTimePicker = genericComponent<VTimePickerSlots>()({
           break
       }
 
-      const emitChange = selecting.value === (props.useSeconds ? SelectingTimes.Second : SelectingTimes.Minute)
-
+      const emitChange = selecting.value === (props.useSeconds ? SelectingTimes.Second : SelectingTimes.Minute) ||
+        (SelectingTimes.Hour && (inputMinute.value === 0 || !!inputMinute.value))
```

Comment:
adding `SelectingTimes.Hour` is unnecessary cause it's a constants

`(inputMinute.value === 0 || !!inputMinute.value)` => can this be changed to `inputMinute.value !== null`?

Also, i assume that we need to emitValue whenever all of the `inputHour`, `inputMinute`, `inputSecond` is not null, so could this condition be changed to 
`const emitChange = inputHour.value !== null && inputMinute.value !== null && (props.useSeconds ? inputSecond.value !== null : true)`?

Reply:
Changed.